packages/typespec-vscode/src/vscode-cmd/emit-code/emitter.ts

```diff
@@ -23,7 +23,7 @@ export const PreDefinedEmitterPickItems: Record<string, vscode.QuickPickItem> =
     detail: "Emitting OpenAPI3 Document from TypeSpec files.",
   },
   client: {
-    label: "Client Code",
+    label: "<PREVIEW> Client Code",
```

Comment:
Please separate this into a different PR. I think we still need more discussion on this. I would prefer to remove the <PREVIEW> tag which is hard to maintain to make sure it's up-to-date when we can only hardcode it just like ```tsp init```.